kumascript/macros/HTMLRef.ejs

```diff
@@ -1,69 +1,61 @@
 <%
-function containsTag(tagList, tag) {
-    if (tagList == null || tagList == undefined) return 0;
-    if (tag == undefined || tag == null) return 0;
-    tag = tag.toLowerCase();
-    for (var i = 0, len = tagList.length; i < len; i++) {
-        if (tagList[i].toLowerCase() == tag) return 1;
-    }
-    return 0;
+function containsTag(tagList, matchTag) {
+  if (!tagList || !matchTag) return 0;
+  matchTag = matchTag.toLowerCase();
+  for (const tag of tagList) {
+    if (tag.toLowerCase() == tag) return 1;
+  }
+  return 0;
 }
 
-var s_html_href = '/'+env.locale+'/docs/Web/HTML';
-var s_html_ref_href = '/'+env.locale+'/docs/Web/HTML/Element';
-var s_html_ref_title = 'HTML Elements';
-switch (env.locale) {
-    case 'fr':
-        s_html_href = '/'+env.locale+'/docs/HTML';
-        s_html_ref_href = '/'+env.locale+'/docs/Web/HTML/Element';
-        s_html_ref_title = 'Éléments HTML';
-        break;
-    case 'ru':
-        s_html_ref_title = 'HTML элементы';
-        break;
-    default: break;
-}
+const s_html_ref_href = `/en-US/docs/Web/HTML/Element`;
```

Comment:
see line 43, if use a localized slug here, we may not got the subpages. And find the subpages in `en-US` could show all the subpages (fallback to en-US  if not localized)

![image](https://user-images.githubusercontent.com/15844309/198797518-0b573aaa-ae41-4844-8a36-c3b22e0e025b.png)